content/events/22-23/sss-codenight.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Student Street Soccer Scoreboard app Codenight'
+description: 'We zullen het Student Street Soccer event van VEK voor eens en voor altijd een Zeus-project maken!'
+created_at: 09-04-2023
+time: '13-04-2023 18:00'
+end: '13-04-2023 23:00'
+banner: 'https://pics.zeus.gent/R1vArdGr174blVDUPnJg0bgQosAV0Qyj5BR9gXHB.jpg'
+header_text_background: true
+location: 'De Kelder'
+locationlink: $kelder
+---
+
+Omdat we jullie tijdens dit paasreces geen twee weken kunnen missen en omdat we de kans hebben om opnieuw van iets een Zeus-project te maken organiseren we komende **donderdag 13/04 vanaf 18:00** een codenight!
```

Comment:
- s/dit/deze
- 2de omdat (achter en) weglaten

Reply:
Te laat, kzal het wel aanpassen in mijn MM aankondiging